### DISCUSSION TITLE
Is there a way to filter only by script and not apps in /Applications?

### DISCUSSION THREAD

**Discussion by ramiroaraujo** [johnlindquist/kit#1315] [Q&A] [answered]
Hey community! simple question. I use ScriptKit only for the actual scripts, and (for the moment at least) keep using Alfred for everything else. 
I noticed that in some of the latest updates regular Apps are popping in in the results when filtering. Is there a way to avoid this? :)

Thanks!

**Comment by johnlindquist** [comment, marked as answer]
@ramiroaraujo 

In your `~/.kenv/.env` add this line:

```bash
KIT_EXCLUDE_KENVS=Apps
```

Although "Apps" isn't technically a "kenv", treated as a kenv group in the main menu.

You can exclude multiple kenvs from the main menu with a comma-separated list:

```bash
KIT_EXCLUDE_KENVS=Apps,examples
```

**Reply by ramiroaraujo** [reply]
rather late to check for the answer, but THANKS! :)
